stable/clamav/templates/daemonset.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.daemonset.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "clamav.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ template "clamav.name" . }}
+    helm.sh/chart: {{ template "clamav.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "clamav.name" . }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "clamav.name" . }}
+    spec:
+      tolerations:
+      - key: node-role.kubernetes.io/master
```

Comment:
What about if you run pods on your master? this should be something in the values commented, so if you want, just uncoment this lines and they will be included here.